Source/Request.swift

```diff
@@ -118,7 +118,7 @@ class Request
 			"version" : MESSAGE_VERSION ]
 		do {
 			let data = try JSONSerialization.data(withJSONObject: msgDict, options: JSONSerialization.WritingOptions(rawValue: 0))
-			let string = NSString(data: data, encoding: String.Encoding.utf8.rawValue) as? String
+			let string = NSString(data: data, encoding: String.Encoding.utf8.rawValue) as String?
```

Comment:
These are not technically the same thing. as String? can result in crashes if the value exists but does not cleanly convert to a string. However, that NSString initializer should only return an NSString or nil, so this is probably safe.